holoviews/plotting/bokeh/raster.py

```diff
@@ -303,8 +303,7 @@ def get_data(self, element, ranges, style):
 
     def _hover_opts(self, element):
         xdim, ydim = element.kdims
-        print(xdim)
-        super()._hover_opts(element)
+        return [(xdim.pprint_label, '$x'), (ydim.pprint_label, '$y')], {}
```

Comment:
I would like to see a comment about why it does not work right now and what it needed to get it to work in the future.